tests/suites/cli/active_branch.sh

```diff
@@ -0,0 +1,88 @@
+# Checks juju indicates the correct branch output tabular and json.
```

Comment:
Because these are features (branches etc) and because I don't want "cli" to get massive. So can we move it to the branches suite. https://github.com/juju/juju/tree/develop/tests/suites/branches

thx